contentcuration/contentcuration/frontend/shared/views/MainNavigationDrawer.vue

```diff
@@ -120,6 +123,10 @@
         type: Boolean,
         default: false,
       },
+      color: {
+        type: String,
+        default: 'secondary',
+      },
```

Comment:
I added a `color` prop here as there are instances where the `MainNavigationDrawer` top needs to match with the `AppBar` color since its not always `yellow`. We actually do have a white app bar too and this prop provides the much needed flexibility.